module-4/backend/week_one.md

```diff
@@ -5,16 +5,44 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+  -General syntax and structure of Javascript. 
+
 2. What are some tools to help debug JavaScript code?
+  -Chrome Dev Tools
+  -debugger
+  -console.log(thing)
+
 3. What are some tools you need in order to unit test your JavaScript?
+  -npm
+  -mocha
+  -chai
+
 4. What is the syntax for invoking a function?
-5. What's the difference between `==` and `===` in JavaScript?
-6. What's the difference between asynchronous and synchronous JavaScript? 
-7. How do we setup a route when creating an API with Node and Express?
-8. What do we use Knex for?
-9. What's `npm` and what do we use it for?
+  -functionName()
+
+5. What's `this` in JavaScript?
+  -'this' is a way of calling a particular 'thing' (for lack of a better term) on the page. 'This' could be an ajax call, an object, a tag or element. It depends on where 'this' is called.
+
+6. What is Webpack and why is it useful?
+  -Webpack bundles and readies javascript code (minifies and uglifies) for use in your development environment.
+
+7. When/why do you want to use event delegation?
+  -Event delegation can be used for targeting elements that may exist inside of parent elements. This is useful if you have multiple elements with the same name/id/class that exist in distinct parent elements.
```

Comment:
it's also useful for selecting elements that aren't on the page on document ready!